recipes/libpq/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package C)
+project(test_package LANGUAGES C) # if the project is pure C
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(PostgreSQL REQUIRED)
+find_package(PostgreSQL REQUIRED CONFIG)
```

Comment:
```suggestion
find_package(PostgreSQL REQUIRED)
```

Reply:
@SpaceIm, removing the CONFIG will break the test_v1_package tests since CMake will try to use `FindPostgreSQL.cmake` module which won't find the conan library.

Reply:
because you have changed generator in test v1 package

Reply:
I haven't. It's the same as https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/autotools_package/all/test_v1_package/conanfile.py

Anyway, that isn't the issue. I've tested the suggested change and it doesn't work on Linux GCC. Can you test on your setup?

Reply:
you have change what was originally in test package and moved to test v1 package.

Reply:
I haven't https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/autotools_package/all/test_package/CMakeLists.txt

Reply:
I'm talking about libpq recipe, not templates which are just suggestions.

Reply:
I've made the update.